xpk.py

```diff
@@ -3475,7 +3910,37 @@ def directory_path_type(value):
         'Defaults to 30 seconds.'
     ),
 )
-
+workload_pathways_workload_arguments.add_argument(
+    '--use-pathways', 
+    action='store_true',
+    help=(
+        'Provide this argument to create Pathways workloads.'
+    ),
+)
+workload_pathways_workload_arguments.add_argument(
+    '--proxy-server-image', 
+    type=str,
+    default='gcr.io/cloud-tpu-multipod-dev/pathways-prototype/gke/pathways-demo:proxy_server',
+    help=(
+        'Please provide the proxy server image for Pathways.'
+    ),
+)
+workload_pathways_workload_arguments.add_argument(
+    '--server-image', 
```

Comment:
@shaurya89 Any thoughts on whether we should provide default images or have users provide the same (i.e mark this argument as "required")?

Reply:
For now defaults contain images in cloud-tpu-v2-images